datumaro/cli/commands/describe_downloads.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import contextlib
+import sys
+from typing import Dict, Type
+
+from datumaro.components.extractor_tfds import (
+    AVAILABLE_TFDS_DATASETS,
+    TFDS_EXTRACTOR_AVAILABLE,
+    TfdsDatasetRemoteMetadata,
+)
+from datumaro.util import dump_json
+
+from ..util import MultilineFormatter
+
+
+def build_parser(
+    parser_ctor: Type[argparse.ArgumentParser] = argparse.ArgumentParser,
+):
+    parser = parser_ctor(
+        help="Print information about downloadable datasets",
+        description="""
+        Reports information about datasets that can be downloaded with the
+        "datum download" command. The information is reported either as
+        human-readable text (the default) or as a JSON object.
+        """,
+        formatter_class=MultilineFormatter,
+    )
+
+    parser.add_argument(
+        "--report-format",
+        choices=("text", "json"),
+        default="text",
+        help="Format in which to report the information (default: text)",
+    )
+    parser.add_argument(
+        "--report-file", help="File to which to write the report (default: standard output)"
+    )
+    parser.set_defaults(command=describe_downloads_command)
+
+    return parser
+
+
+def get_sensitive_args():
+    return {
+        describe_downloads_command: [],
```

Comment:
```suggestion
        describe_downloads_command: ["report-file"],
```

Reply:
Fixed, thanks.

(As an aside, maybe we should switch to listing safe arguments rather than sensitive ones. It's far too easy to forget to add an argument to the list and commit a privacy violation.)

Reply:
Hmm, it is a good idea.